src/DotNetty.Codecs/DotNetty.Codecs.csproj

```diff
@@ -84,7 +86,5 @@
   <Target Name="AfterBuild">
   </Target>
   -->
-  <ItemGroup>
-    <Folder Include="Strings\" />
-  </ItemGroup>
```

Comment:
can you pls verify that it doesn't affect placement of Strings\StringEncoder.cs and Strings\StringDecoder.cs in project structure? Should be under Strings Folder in project.